src/inputs/ObjectSearch.tsx

```diff
@@ -52,16 +101,75 @@ class ObjectSearch extends Component<IObjectSearchProps> {
       ;
 
     this.search = value;
-    this.props.onSearchChange(this.search);
+    this.isLoading.setTrue();
+    onSearchChange(this.search);
     const response = await getEndpoint(`/${endpoint}/${toKey(params)}`);
     this.options = response.results;
+    this.isLoading.setFalse();
+  }
+
+  private renderOptionAdd () {
```

Comment:
@rkuykendall thoughts ?

Reply:
I think we should consider that as it's own feature separate from this PR. If it's pressing, we should make an issue. Otherwise I think just when it comes up.

Reply:
_( that would be the REAL `ObjectSearch` )_